docs/changelog/97463.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 97463
+summary: "[Enterprise Search] Add connectors indices and ent-search pipeline"
+area: Application
```

Comment:
Is this going to be a standalone feature? If this is part of something else, consider adding the `>non-issue` label so it's not added to the `changelog`